Dockerfile

```diff
@@ -1,3 +1,21 @@
-FROM nginxinc/nginx-unprivileged:mainline
-USER nginx
-COPY ./dist /usr/share/nginx/html
\ No newline at end of file
+FROM node:24-alpine AS builder
+
+WORKDIR /app
+
+RUN corepack enable
+
+COPY pnpm-lock.yaml package.json .
+RUN pnpm fetch
+
+COPY src src
+COPY public public
+COPY *.json *.ts *.cjs *.mts *.html .
+RUN pnpm install
+RUN pnpm build
+
+FROM caddy:2-alpine
```

Comment:
Motivation to switch the base image? We should keep to use unprivileged rootless images

Reply:
Switched to Caddy from NGINX in order to pass `BACKEND_HOST` and `BACKEND_PORT` envs for variable backend reverse proxy setup.

> We should keep to use unprivileged rootless images

Noted, applied changes - no Caddy runs rootless.